.gitlab-ci.yml

```diff
@@ -1,4 +1,13 @@
 stages:
   - benchmarks
 
-include: ".gitlab/benchmarks.yml"
+macrobenchmarks:
+  stage: benchmarks
+  needs: []
+  trigger:
+    include: ".gitlab/benchmarks.yml"
+  rules:
+    - if: $NIGHTLY_BENCHMARKS || $CI_COMMIT_REF_NAME == "master" || $CI_COMMIT_REF_NAME =~ /^release\/v/
+      when: always
+    - when: manual
+  allow_failure: true
```

Comment:
👋  Hey Louis, LGTM
I just have a small nit-pick, you might not want `allow_failure` to be set to `true `in order to better catch broken runs early on before we are close to a release (and have a better history to track potential regressions 😉 )
I think @igoragoli should review the PR since he has more insight on the pre-release quality gates

Reply:
+1

Reply:
addressed in 271fe55